tests/app/filter.test.js

```diff
@@ -0,0 +1,34 @@
+import { expect, test } from "@playwright/test";
+import { loadMap, getNumCities } from "./utils";
+
+test("scope filter updates markers", async ({ page }) => {
+  await loadMap(page);
+
+  const all = await getNumCities(page);
+
+  const checkMarkers = async () => {
+    const cities = await getNumCities(page);
+    return cities > 0 && cities < all;
+  };
+
+  await page.selectOption(".filter--scope", "Regional");
+  expect(await checkMarkers()).toBe(true);
```

Comment:
I think a better test would give a range of numbers for what we expect the value to be. A very big range so that the tests don't start failing when we add new cities. For example, that the number is between 100 and 400. 

Note that the lower bound should be safe: we don't expect to remove cities! We only will add new cities in the future. So if we currently have 104 cities, set the lower bound to 100 and the upper bound to 250, for example.

The issue with this test now is that it doesn't capture nuances like that the behavior is different between the options. You could have a bug where every scope was only showing Regional, and we wouldn't catch that.